OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
This file is required by the [check in OpenShift CI](https://github.com/openshift/release/pull/28308#issuecomment-1117412175)